### PR TITLE
HOTT-4628: Extend Backend ErrorsController

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

<[HOTT-4623](https://transformuk.atlassian.net/browse/HOTT-4628)>

### What?

I have added/removed/altered:

- [ ] Added actions in the controller for the missing error handling actions

### Why?

I am doing this because:

- There are certain exceptions in which are mapped in the default Rails behaviour to actions we don’t currently support
-
-

### Have you? (optional)

- [ ] Added documentation for new apis
- [ ] Added new environment variables with correct values to all apps in all spaces

### Deployment risks (optional)

- Includes destructive/migratory actions which may effect critical data
- Changes an api that is used in production

### CODE REVIEW

File: spec/requests/errors_controller_spec.rb [hunk 1]
```diff
@@ -75,4 +75,28 @@
   describe 'GET #serice_unavailable' do
     it_behaves_like 'a csv or json error response', 503, 'Service is unavailable'
   end
+
+  describe 'GET #maintenance' do
```
Comment (alexdesi) [inline spec/requests/errors_controller_spec.rb:79]: `def maintenance` didn't have a spec.

**Review by willfish — DISMISSED** [review]
LGTM

File: spec/requests/errors_controller_spec.rb (hunk 1 above)
Comment (jebw) [reply spec/requests/errors_controller_spec.rb:79]: We've already got a separate feature spec for maintenance mode so this isn't required, see

`spec/features/maintenance_mode_spec.rb`

**Review by jebw — CHANGES_REQUESTED** [review]
Looks good but we have already got a separate feature spec for maintenance mode?

**Review by willfish — APPROVED** [review]

**Review by willfish — APPROVED** [review]

**Comment by willfish** [comment]
The feature spec for the maintenance mode is a bit different from the plumbing tests of the errors even though it piggy backs off of the same errors

**Pull request merged**